app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+  def index
+  end
+end
```

Comment:
Resolved: this now is the default route and the file is necessary (tested by `rm`ing and `rake test`ing).